js/image/Index.svelte

```diff
@@ -64,8 +62,12 @@
 		share: ShareData;
 	}>;
 
-	$: url = _value?.url;
-	$: url, gradio.dispatch("change");
+	$: {
```

Comment:
why not just listen to the url change as before?

Reply:
Yeah that would be okay too, I ended up removing quite a few things from Image, including this line, and then I copied this snippet over from the Audio component. Either way is fine I think